### DISCUSSION TITLE
UART bootloader without DTR/RTS.

### DISCUSSION THREAD

**Discussion by r1000ru** [ExpressLRS/ExpressLRS#1342] [Ideas]
Hello.There is a proposal to make it possible to update by UART0 without using DTR and RTS. I.e. by sending a command to receive the firmware, after that-the transfer of the firmware and then-restart (as by WiFi). This can be useful for homemade transmitters modules used for development that do not have DTR/RTS inverters on board with subsequent connection to Boot and Reset.

*Instead of commenting "I want this too", consider leaving a :+1: instead, to reduce spam and clutter*

**Comment by grc1000** [comment]
would that solve the problem of the EP2 with solid LED when removed from the package? new receiver never managed to update.
